packages/neon/neon/lib/src/blocs/next_push.dart

```diff
@@ -27,19 +27,15 @@ class NextPushBloc extends Bloc implements NextPushBlocEvents, NextPushBlocState
     Rx.merge([
       _globalOptions.pushNotificationsEnabled.stream,
       _globalOptions.pushNotificationsDistributor.stream,
-      _globalOptions.pushNotificationsDistributor.values,
       _accountsBloc.accounts,
     ]).debounceTime(const Duration(milliseconds: 100)).listen((final _) async {
-      if (!_globalOptions.pushNotificationsEnabled.enabled.hasValue ||
-          !_globalOptions.pushNotificationsEnabled.enabled.value ||
-          !_globalOptions.pushNotificationsEnabled.hasValue ||
-          !_globalOptions.pushNotificationsEnabled.value) {
+      if (!_globalOptions.pushNotificationsEnabled.enabled || !_globalOptions.pushNotificationsEnabled.value) {
```

Comment:
I remember this was a lot of pain, needs to be thoroughly tested

Reply:
Before enabling push notifications it showed me a warning that no UnifiedPush distributor was detected. After enabling it the warning was gone and the distributors were also present in the list.

Reply:
The warning also appears again when disabling the push notifications. Other than that everything seems to work as expected!